### DISCUSSION TITLE
Error: "time limit exceeded" when using lego with zeroSSL.

### DISCUSSION THREAD

**Discussion by ngochip** [go-acme/lego#1619] [Q&A]
> [root@vps accounts]# /usr/local/bin/lego --path /usr/local/directadmin/data/.lego --dns.resolvers 1.1.1.1 --accept-tos -s https://acme.zerossl.com/v2/DV90 --cert.timeout 300 -m admin@test.com --http --http.webroot /var/www/html -d test.com --key-type rsa4096 run --no-bundle
> 
> 2022/04/10 16:02:34 [INFO] [test.com] acme: Obtaining SAN certificate
> 2022/04/10 16:02:46 [INFO] [test.com] AuthURL: https://acme.zerossl.com/v2/DV90/authz/JllX0qXM-GMR3psZVxYVQA
> 2022/04/10 16:02:46 [INFO] [test.com] acme: use http-01 solver
> 2022/04/10 16:02:46 [INFO] [test.com] acme: Trying to solve HTTP-01
> 2022/04/10 16:02:55 [INFO] [test.com] The server validated our request
> 2022/04/10 16:02:55 [INFO] [test.com] acme: Validations succeeded; requesting certificates
> 2022/04/10 16:03:03 [INFO] Wait for certificate [timeout: 5m0s, interval: 5s]
> 
> 2022/04/10 16:08:07 Could not obtain certificates:
>         error: one or more domains had a problem:
> [test.com] time limit exceeded
>

I use LEGO with zeroSSL. however the last few days i am not able to generate new certificate.
I tried increasing the timeout from 30s to 5m. But it doesn't seem to work.

my domain is still active (I changed it to test.com in the above).

Please let me know what needs to be done to solve this?

thanks.

**Comment by epsilon-0** [comment]
Having the same problem, with wildcard certificates and dns-01 challenge.
